worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -433,6 +435,24 @@ public void snow(Player player, LocalSession localSession,
             new ApplyLayer(new Snow(stack)), shape, "worldedit.brush.snow");
     }
 
+    @Command(
+        name = "biome",
+        desc = "Biome brush, sets biomes in the area"
+    )
+    @CommandPermissions("worldedit.brush.biome")
+    public void biome(Player player, LocalSession localSession,
+                      @Arg(desc = "The shape of the region")
+                          RegionFactory shape,
+                      @Arg(desc = "The size of the brush", def = "5")
+                          double radius,
+                      @Arg(desc = "The biome type")
+                          BiomeType biomeType) throws WorldEditException {
+
+        setOperationBasedBrush(player, localSession, radius,
+            new ApplyRegion(new Biome(biomeType)), shape, "worldedit.brush.biome");
+        player.printInfo(TranslatableComponent.of("worldedit.setbiome.warning"));
```

Comment:
potentially this message should also be applied to `regen -b` and `paste/stack/move -b` (though it doesn't directly concern this PR but now that the message is its own key...)